ansible/roles/dokuwiki/tasks/main.yml

```diff
@@ -21,39 +21,38 @@
   stat:
     path: /opt/dokuwiki
   register: dokuwiki
-  tags:
-    - prerequisites
 
-- when: dokuwiki.stat.exists == False
+- when: dokuwiki.stat.exists == False or dokuwiki.stat.isdir == True
```

Comment:
What is the point of this conditional? It's always giong to be true.

Reply:
If the thing doesn't exist, then we install it. If it's a directory, that means we're on a bad prerequisites build that has already created one empty dir there, and nothing else.

The conditional is False if `/opt/dokuwiki` is a link, which means it's a link to `/var/lib/liquid/dokuwiki`, which means it's already installed and shouldn't be installed again.

Reply:
Right, makes sense now. Please add a comment explaining this.